perception/shape_estimation/src/node.cpp

```diff
@@ -44,7 +45,10 @@ ShapeEstimationNode::ShapeEstimationNode(const rclcpp::NodeOptions & node_option
   bool use_corrector = declare_parameter("use_corrector", true);
   bool use_filter = declare_parameter("use_filter", true);
   use_vehicle_reference_yaw_ = declare_parameter("use_vehicle_reference_yaw", true);
-  estimator_ = std::make_unique<ShapeEstimator>(use_corrector, use_filter);
+  std::string bbox_optimizer_str = declare_parameter("bbox_optimizer", "original");
+  bbox_optimizer_ =
+    (bbox_optimizer_str == "boost" ? BoundingBoxOptimizer::Boost : BoundingBoxOptimizer::Original);
+  estimator_ = std::make_unique<ShapeEstimator>(use_corrector, use_filter, bbox_optimizer_);
```

Comment:
```suggestion
  std::string bbox_optimizer_str = declare_parameter("bbox_optimizer", "original");
  const auto bbox_optimizer =
    (bbox_optimizer_str == "boost" ? BoundingBoxOptimizer::Boost : BoundingBoxOptimizer::Original);
  estimator_ = std::make_unique<ShapeEstimator>(use_corrector, use_filter, bbox_optimizer);
```
Also `bbox_optimizer_` can be removed.
https://github.com/autowarefoundation/autoware.universe/pull/1292/files#diff-dbbbd08e59261a1033fea525cb398d053d84deb04d5f0651ca20eff57f48b791R39

Reply:
Could also implement a proper if/else switch on `bbox_optimizer_str` and report an error or stop the node if the string is neither "boost" nor "original"?

Reply:
I think it's ok to select "original" with `ROS_INFO("select original")`, if the string is neither "boost" nor "original".
If these are the only two options, you may as well use bool `use_fast_bbox_optimizer`.

Reply:
Changed this param to a `bool` as suggested.

Reply:
Thank you! LGTM